src/matplotlib/stylelib/dune.mplstyle

```diff
@@ -45,7 +45,7 @@ axes.prop_cycle: cycler('color', ['000000', 'D55E00', '56B4E9', 'E69F00', '009E7
 #axes.prop_cycle: cycler('color', ['000000', 'D55E00', '0072B2', 'E69F00', '009E73', 'CC79A7', '56B4E9', 'F0E442',])
 
 # We recommend Helvectica (consistent with ROOT).  The others are Helvetica "clones" in case Helvetica isn't directly available
-font.sans-serif:  Helvetica, Helvetica Neue, Nimbus Sans, Open Sans
+font.sans-serif:  Helvetica, Helvetica Neue, Nimbus Sans, Liberation Sans, Arial
```

Comment:
Liberation and Arial are quite different from Helvetica in my opinion.

Another clone available on systems with LaTeX could be TeX Gyre Heros, which is very similar to Helvetica.

Reply:
Yeah, they're certainly in priority order here, and the hope would be that we never get down to the bottom.  FWIW, according to Wikipedia, Arial was explicitly designed to function like Helvetica (and Liberation Sans is an Arial knock-off).  The article includes this sentence, which you can decide for yourself on 😅:

> Because of their almost identical appearances, both Arial and Helvetica have commonly been mistaken for each other. 

Also FWIW Gyre Heros is not installed on the GPVMs, which I'd guess would be the place most likely to not have Helvetica or Nimbus but to have Liberation...

Reply:
I debated whether it was worth shipping our own copy of a Helvetica font, but I decided that was far too much trouble :smile:

Reply:
> Yeah, they're certainly in priority order here, and the hope would be that we never get down to the bottom. FWIW, according to Wikipedia, Arial was explicitly designed to function like Helvetica (and Liberation Sans is an Arial knock-off). The article includes this sentence, which you can decide for yourself on sweat_smile:
> 
> > Because of their almost identical appearances, both Arial and Helvetica have commonly been mistaken for each other.
> 
> Also FWIW Gyre Heros is not installed on the GPVMs, which I'd guess would be the place most likely to not have Helvetica or Nimbus but to have Liberation...

Ok, I concede :D